browser/ui/webui/playlist_ui.cc

```diff
@@ -89,7 +89,7 @@ class UntrustedPlayerUI : public ui::UntrustedWebUIController {
     source->OverrideContentSecurityPolicy(
         network::mojom::CSPDirectiveName::MediaSrc,
         std::string("media-src 'self' chrome-untrusted://playlist-data "
-                    "https://*.googlevideo.com:*;"));
+                    "https://*:* http://localhost:*;"));
```

Comment:
i think you can simplify this to just `https:` instead of `https://*:*` ? https://content-security-policy.com/#source_list

Reply:
Yes, right! Changed